app/src/main/java/net/skyscanner/backpack/demo/BackpackDemoApplication.kt

```diff
@@ -0,0 +1,14 @@
+package net.skyscanner.backpack.demo
+
+import android.support.multidex.MultiDexApplication
+import android.support.v7.app.AppCompatDelegate
+
+/**
+ * Application class registered in AndroidManifest.xml
+ */
+class BackpackDemoApplication : MultiDexApplication() {
+  override fun onCreate() {
+    super.onCreate()
+    AppCompatDelegate.setCompatVectorFromResourcesEnabled(true)
```

Comment:
wasn't this `vectorDrawables.useSupportLibrary = true` supposed to make it work with the api 19?

Reply:
Surprisingly its both. Adding SO link to PR for reference.